apps/docs/content/components/forms/Checkbox.mdx

```diff
@@ -1,48 +1,134 @@
 ---
-title: Checkbox (WIP)
-description: TBD
+title: Checkbox
+description: Checkboxes are used when there are multiple items to select in a list. Users can select zero, one, or any number of items.
 category: "forms"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/checkbox/src/Checkbox.tsx
 ---
 
+<Example src="checkbox/docs/checkbox/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+### Checkbox
+
+<PropTable component="Checkbox" />
+
+### CheckboxField
 
-## Guidelines
+<PropTable component="CheckboxField" />
 
-TODO: If we have some guidelines about this component's usage
+<FeatureFlag flag="next">
+    ## Guidelines
 
-### Accessibility ?
+    TODO: If we have some guidelines about this components usage
 
-TODO: If we have some guidelines about this component and accessibility
+    ### Accessibility ?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
 ## Anatomy
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+<FeatureFlag flag="rc">
+
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+- [Icon](./Icon)
+- [IconList](./IconList)
+- [Text](./Text)
+
+## Checkbox Examples
+
+### No Label
+
+A Checkbox can be rendered without a label.
+
+<Example src="checkbox/docs/checkbox/nolabel"/>
+
+### Checked
+
+A Checkbox can be checked.
+
+<Example src="checkbox/docs/checkbox/checked"/>
+
+### Indeterminate
+
+A Checkbox can be indeterminate.
+
+<Example src="checkbox/docs/checkbox/indeterminate"/>
+
+### Disabled
+
+A Checkbox can be disabled.
+
+<Example src="checkbox/docs/checkbox/disabled" />
+
+### Read-only
+
+A Checkbox can be read-only.
+
+<Example src="checkbox/docs/checkbox/readonly" />
+
+### Invalid
+
+A Checkbox can be invalid.
+
+<Example src="checkbox/docs/checkbox/invalid"/>
+
+### Sizes
+
+A Checkbox can be rendered in different sizes.
+
+<Example src="checkbox/docs/checkbox/size"/>
+### Description
```

Comment:
This should be in the checkbox field examples. The point of the field is to be able to add a description.
I would replace the first checkboxField example with this.